pkg/hub/managedclusterset/default_managedclusterset_label_controller.go

```diff
@@ -0,0 +1,101 @@
+package managedclusterset
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
+	clientset "open-cluster-management.io/api/client/cluster/clientset/versioned"
+	clusterinformerv1 "open-cluster-management.io/api/client/cluster/informers/externalversions/cluster/v1"
+	clusterlisterv1 "open-cluster-management.io/api/client/cluster/listers/cluster/v1"
+	clusterv1 "open-cluster-management.io/api/cluster/v1"
+)
+
+const (
+	defaultManagedClusterSetValue = "default"
+)
+
+// defaultManagedClusterSetLabelController reconciles ManagedClusterSet label.
```

Comment:
Add a comment here to indicate that the purpose of this controller is to add clusterset of the old version to the default clusterset if it has not cluster label. And this controller should be removed in the next release

Reply:
done. https://github.com/open-cluster-management-io/registration/pull/202/files#diff-c991f1a201d952eec15d4a0796413eae3706e1e41ea4144640fefb7e82b554faR25-R28